sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 This version of the SDK defaults to the latest supported service version, which currently is v1.0
 
+**Breaking changes**
+
+- `create_translation_job` was renamed to `begin_translation` and now returns a `DocumentTranslationPoller`
+(or `AsyncDocumentTranslationPoller`) to begin the long-running operation. Call `.result()` on the poller object
+to wait until the translation is complete. See the README for more information about LROs.
+- Upon completion of the LRO, `begin_translation` now returns a pageable of `DocumentStatusResult`. All job-level metadata can still
+be found on `poller.details`.
+- `has_completed` has been removed from `JobStatusResult` and `DocumentStatusResult`. Use `poller.done()` to check if the 
```

Comment:
ohhh interesting. So if a user wants to know if a specific document is done, they will have to check the status of the document and "calculate it" or just wait until the poller finishes, right?
why?

Reply:
People found this confusing in the user study - they thought it was a dynamic property that would update the status in python. Moving to the poller allows us to actually have that dynamic property with `done()`. I feel it's less important to have on DocumentStatusResult, at that point you're actually interested in checking what the status is - "failed", "succeeded" etc to see what to do (at least that's how all our samples are written). Checking with johan on this one.

Reply:
Johan said to remove it